ros2/test.py.tpl

```diff
@@ -18,9 +18,12 @@ sys.argv = sys.argv[:1] + [
 ] + sys.argv[1:]
 
 test_outputs_dir = os.environ.get('TEST_UNDECLARED_OUTPUTS_DIR')
+if test_outputs_dir is None:
+    test_outputs_dir = os.environ.get('TEST_TMPDIR')
 if test_outputs_dir:
```

Comment:
This is not needed as per https://bazel.build/reference/test-encyclopedia#initial-conditions TEST_TMPDIR is always available.

Reply:
good point!